lib/internal/fs/watchers.js

```diff
@@ -174,6 +174,8 @@ FSWatcher.prototype[kFSWatchStart] = function(filename,
   }
 };
 
+FSWatcher.prototype.start = () => {};
```

Comment:
Is it worth adding a comment explaining that this is to maximize backward-compatibility for end users but this function should not be documented? Otherwise, someone else might come along later and remove it as unused or something?

Reply:
fair point.   I'll add a little comment